src/monodroid/jni/monodroid-glue.cc

```diff
@@ -368,14 +370,41 @@ thread_end (MonoProfiler *prof, uintptr_t tid)
 
 FILE *jit_log;
 
+static void
+jit_begin (MonoProfiler *prof, MonoMethod *method)
+{
+	char *name;
+
+	jit_time.mark_end ();
+
+	if (!jit_log)
+		return;
+
+	name = monoFunctions.method_full_name (method, 1);
```

Comment:
This should just be:

```suggestion
	char *name = monoFunctions.method_full_name (method, 1);
```

And remove the `char *name` declaration on line 376.

Reply:
`const char*` would be preferred :)